README.md

```diff
@@ -18,314 +16,247 @@ Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see addi
-
-## Getting Started
-
-### Python setup
-
-Requires an installation of Python 3.7+.
+![DALL·E 2023-01-10 15 19 31](https://user-images.githubusercontent.com/15331990/211682578-3e341709-6509-4c95-a6af-3b1160fe2961.png)
 
-We recommend using a Python virtual environment manager such as pipenv, conda, or virtualenv.
-
-These tasks are designed to work with Prefect 2.0. For more information about how to use Prefect, please refer to the [Prefect documentation](https://orion-docs.prefect.io/).
-
-### Installation
-
-Install `prefect-dask` with `pip`:
-
-```bash
-pip install -U prefect-dask
-```
-
-## Running tasks on Dask
-
-The `DaskTaskRunner` is a parallel task runner that submits tasks to the [`dask.distributed`](http://distributed.dask.org/) scheduler. 
-
-By default, a temporary Dask cluster is created for the duration of the flow run.
-
-For example, this flow counts up to 10 in parallel (note that the output is not sequential).
-
-```python
-import time
-
-from prefect import flow, task
-from prefect_dask import DaskTaskRunner
-
-@task
-def shout(number):
-    time.sleep(0.5)
-    print(f"#{number}")
+Visit the full docs [here](https://PrefectHQ.github.io/prefect-dask) to see additional examples and the API reference.
 
-@flow(task_runner=DaskTaskRunner)
-def count_to(highest_number):
-    for number in range(highest_number):
-        shout.submit(number)
+The `prefect-dask` collection makes it easy to include distributed processing for your flows. Check out the examples below to get started!
 
-if __name__ == "__main__":
-    count_to(10)
+## Integrate with Prefect flows
 
-# outputs
-#3
-#7
-#2
-#6
-#4
-#0
-#1
-#5
-#8
-#9
-```
+Perhaps you're already working with Prefect flows. It downloads many images to train your machine learning model--the problem is that it runs slow because it's executed sequentially.
 
-If you already have a Dask cluster running, either local or cloud hosted, you can provide the connection URL via an `address` argument.
+With `prefect-dask`, you can parallelize your flow, and the best part is that it's as easy as:
 
-To configure your flow to use the `DaskTaskRunner`:
+1. Adding the import: `from prefect_dask import DaskTaskRunner`
+2. Specifying the task runner in the flow decorator: `@flow(task_runner=DaskTaskRunner)`
+3. Submitting tasks to the flow's task runner: `a_task.submit(*args, **kwargs)`
 
-1. Make sure the `prefect-dask` collection is installed as described earlier: `pip install prefect-dask`.
-2. In your flow code, import `DaskTaskRunner` from `prefect_dask.task_runners`.
-3. Assign it as the task runner when the flow is defined using the `task_runner=DaskTaskRunner` argument.
+Here's the before and after!
 
-For example, this flow uses the `DaskTaskRunner` configured to access an existing Dask cluster at `http://my-dask-cluster`.
+=== "Before"
 
-```python
-from prefect import flow
-from prefect_dask.task_runners import DaskTaskRunner
+    ```python hl_lines="1"
+    # Completed in 15.2 seconds
 
```

Comment:
Maybe adding an collapse/expand box for the code samples would be good. Probably not.

Reply:
I think it's better visible by default